content/en/configuration/coap-gateway.md

```diff
@@ -58,11 +58,12 @@ OAuth2.0 Client is used to issuing an authorization code used by the Onboarding
 
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
-| `api.coap.authorization.ownerClaim` | string | `Claim used to identify owner of the device.` | `"sub"` |
+| `api.coap.authorization.ownerClaim` | string | `Claim used to identify owner of the device. Do not use api.coap.authorization.ownerClaim with "sub" value for api.coap.authorization.providers[].grantType with clientCredentials. Otherwise, all devices will be under the same user.` | `"sub"` |
```

Comment:
We need to prevent this combination programatically. Not documenting it.

Reply:
Fixed in https://github.com/plgd-dev/hub/pull/669

Reply:
Please use the formulation from hub MR.

Reply:
fixed